assets/tests/components/__snapshots__/routeLadder.test.tsx.snap

```diff
@@ -266,61 +266,53 @@ Array [
       viewBox="-92 -55 184 0"
       width={184}
     >
+      <line
+        className="m-ladder__scheduled-line on-time"
+        x1={-63}
+        x2={-40}
+        y1={-110}
+        y2={-110}
+      />
       <g
         className="m-ladder__vehicle  "
         onClick={[Function]}
         transform="translate(63,-27.5)"
       >
         <g
-          className="m-vehicle-icon m-vehicle-icon--medium"
+          className="m-vehicle-icon m-vehicle-icon--medium on-time"
         >
           <rect
             className="m-vehicle-icon__label-background"
             height={11}
             rx={5.5}
             ry={5.5}
-            width={40}
-            x={-20}
+            width={26}
+            x={-13}
             y={11.5}
           />
           <text
-            className="m-vehicle-icon__label m-vehicle-icon__label--extended"
+            className="m-vehicle-icon__label m-vehicle-icon__label--normal"
             dominantBaseline="central"
             textAnchor="middle"
             x="0"
             y={17}
           >
-            14/18
+            1
```

Comment:
This commit changed the text from "14/18" to "1", but I can't figure out why, and it makes me worry that the test isn't showing the crowding data anymore. Any idea what's causing it and if it's expected?

Reply:
Broke the test by leaving out the crowding toggle state, fixed.